Rust/tokio-echo/README.md

```diff
@@ -0,0 +1,91 @@
+# tokio_echo
+
+This demo is a TCP Echo server for tokio
+
+## Prerequisites
+
+Be sure to update your rust stable toolchain to at least 1.60.0.
+
+Please ensure that you're using the standard toolchain (rustup default).
+
+## Running from Try Enarx
+
+First generate the Wasm file on localhost:
+
+```console
+cargo build --target wasm32-wasi 
+```
+
+Access the __Try Enarx__ website:
+
+https://try.enarx.dev
+
+Select a platform. Let's try Intel SGX on Equinix.
+
+Upload the generated Wasm file from:
+
+```console
+/target/wasm32-wasi/debug/tokio-echo.wasm 
```

Comment:
This is not correct, because `/` means root. Instead, it should be stated here that after the build, there should be a `target` directory created and the wasm file should be located there